app/models/queue_item.rb

```diff
@@ -0,0 +1,20 @@
+class QueueItem < ActiveRecord::Base
+  belongs_to :user
+  belongs_to :video
+
+  delegate :category, to: :video
+  delegate :title, to: :video, prefix: :video
+
+  def video_title
+    video.title
+  end
+
+  def rating
+    review = Review.where(user_id: user.id, video_id: video.id).first
```

Comment:
`where(arg).first` should be just `find_by`. In Rails 4, it is the best way to search for 1 record, use `where` to search for a collection.